projects/schematics/src/migrations/5_0/rename-symbol/checkout-rename-symbol.ts

```diff
@@ -1,20 +0,0 @@
-import {
-  SPARTACUS_CHECKOUT_B2B_ROOT,
-  SPARTACUS_SETUP,
-} from '../../../shared/libs-constants';
-import { RenamedSymbol } from '../../../shared/utils/file-utils';
-
-export const CHECKOUT_RENAMED_SYMBOLS_DATA: RenamedSymbol[] = [
-  // core-libs/setup/recipes/b2b/config/default-b2b-checkout-config.ts
-  {
-    previousImportPath: SPARTACUS_SETUP,
-    previousNode: `defaultB2bCheckoutConfig`,
-    newImportPath: SPARTACUS_CHECKOUT_B2B_ROOT,
```

Comment:
Maybe we can mention in the migration comment for this in the `breaking-change.json` that they should use/find it from `@spartacus/checkout/b2b/root` instead.

Reply:
The problem I see with this approach defaultB2bCheckoutConfig is not removed.

I thought rather this kind of documentation was more relevant in the release notes about the feature.  Let's discuss.